crates/services/txpool_v2/src/pool_worker.rs

```diff
@@ -153,6 +158,33 @@ impl PoolWorkerInterface {
             })
     }
 
+    pub fn remove_skipped_transaction(
```

Comment:
Why do we need this function if we can just pass `PreConfirmationSqueezedOut` in the `preconfirmed_transaction`?

Reply:
Done